packages/to-dts/lib/cli.js

```diff
@@ -32,6 +32,11 @@ const toDts = {
         describe: 'File to write to',
         type: 'string',
       },
+      showGeneratedComment: {
```

Comment:
IMO "show" does seem a little bit off. I would think `includeDisclaimer` or `generateDisclaimer` makes more sense. But this is mostly me that associate "show" with something else.

Reply:
`--includeDisclaimer` is superior to `--showGeneratedComment` imo. Let me change that!

Reply:
The name is now updated to `includeDisclaimer`.